typescript/forms.d.ts

```diff
@@ -865,6 +865,9 @@ export type Form = {
    * `postSubmissionActions`.
    */
   redirectUrl?: string
+  postSubmissionReceipt?: {
+    html: string
```

Comment:
I assume this is to cater for additional future properties?

Reply:
yep, PDF generation is coming up next story